docs/src/main/sphinx/connector/query-passthrough-warning.fragment

```diff
@@ -0,0 +1,8 @@
+The native query passed to the underlying data source is required to return a
+table as a result set. This can be a one column table with one row. No
+validation or security checks of these queries are performed by Trino. Only the
+data source performs these tasks using its own configuration. You can use
+passthrough queries to read data; however, queries that update, insert, or
+delete data cannot be used since they do not return a table. If supported by the
+underlying data source, you can enable such queries by appending a statement
+with ``RETURNING 1``.
```

Comment:
Maybe we should add afterwards 

These characteristics can result in undesired data access or data loss.

Or so

Reply:
Yes. That's the kind of warning I wanted to add. Should I create a callout box?

Reply:
No .. just have separate paragraphs and then lets see what reviewers think. Personally I think its fine to just use normal paragraphs.